src/logic/function/basic.lean

```diff
@@ -306,6 +306,9 @@ if h : a = a' then eq.rec v h.symm else f a
 @[simp] lemma update_same (a : α) (v : β a) (f : Πa, β a) : update f a v a = v :=
 dif_pos rfl
 
+lemma injective_update (f : Πa, β a) (a' : α) : injective (update f a') :=
```

Comment:
I think convention might be to name this the other way around

Reply:
Probably you should ask on Zulip. I don't remember what was the outcome of the laste discussion. We definitely have both `injective_*` and `*_injective`.

Reply:
[The last discussion says `update_injective`](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/naming.20of.20injectivity.20lemmas/near/200630369)